### DISCUSSION TITLE
"Field \'user_uuid\' doesn\'t have a default value" when creating an item in an organization on a replicating database.

### DISCUSSION THREAD

**Discussion by x86-39** [dani-garcia/vaultwarden#1343] [Q&A]
Hello, we are trying to set up bitwarden with high availability, this is done by configuring the MySQL server to replicate between the two hosts. We had one other error which was resolved by creating a column in the attachments.akey table. Currently, sync functions between the two hosts, and everything appears to work well so far. However when creating an item in an organization, it fails and this error is visible in the logs
```[2020-11-30 10:28:09.197][request][INFO] POST /api/ciphers/create
[2020-11-30 10:28:09.210][error][ERROR] Error saving cipher.
[CAUSE] DatabaseError(
    __Unknown,
    "Field \'user_uuid\' doesn\'t have a default value",
)
[2020-11-30 10:28:09.210][response][INFO] POST /api/ciphers/create (post_ciphers_create) => 400 Bad Request
``` 
This happens on both hosts. Any ideas on how to fix this issue?

**Comment by BlackDex** [comment]
It seems like something went wrong with the replication or the creation of the database.

Not something we can fix i think.

**Comment by BlackDex** [comment]
Converting this issue to a discussion.
